packages/connex/src/compat.ts

```diff
@@ -16,7 +16,8 @@ export function compat1(connex1: Connex1): Connex {
                         head: {
                             ...s.head,
                             gasLimit: 20000000
-                        }
+                        },
+                        finalized: '0000000000000000000000000000000000000000000000000000000000000000'
```

Comment:
take `t1.genesis.id`?

Reply:
I want to clearly show `finalized` is not supported in compatible mode.